ui/ui.js

```diff
@@ -273,29 +273,21 @@ function runsim(currentGear, fullLogs) {
     priout.innerHTML = metricHTML;
     aiout.innerHTML = metricHTML;
 
-    var veryMax = 0.0;
+    var bestIterMaxDps = 0.0;
```

Comment:
you want to just remove this variable and its use below entirely? It shouldn't be needed now that 'Peak' reports the actual max instead of going down with mana regen.

Reply:
Done